packages/thirdweb/src/react/web/ui/components/Tabs.tsx

```diff
@@ -16,19 +17,33 @@ export default function Tabs({
   const theme = useCustomTheme();
   return (
     <Container>
-      <Container flex="row" center="y" style={{ width: "100%" }}>
+      <Container
+        flex="row"
+        center="y"
+        style={{ width: "100%", borderRadius: radius.md, marginTop: 8 }}
+        p="xxs"
+        bg="secondaryButtonBg"
+      >
         {options.map((option) => (
           <button
             type="button"
             key={option}
             onClick={() => onSelect(option)}
             style={{
               flex: 1,
-              height: "50px",
+              height: "40px",
               display: "flex",
               alignItems: "center",
               justifyContent: "center",
               position: "relative",
+              borderRadius: radius.md,
+              color: option === selected ? theme.colors.primaryText : "auto",
+              backgroundColor:
+                option === selected ? theme.colors.modalBg : "transparent",
+              boxShadow:
+                option === selected
+                  ? `0px 0px 10px ${theme.colors.modalBg}`
```

Comment:
Can we remove this shadow? We are not using shadows anywhere else - so this stands out too much